### PR TITLE
[SonarQube] Support user token authentication

### PR DESCRIPTION
**Pull request by kakawait**
new secret: `sonarqube_token`

Sonarqube does not need user/pass couple, token is enough

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -834,7 +834,18 @@ camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/(.*)\.(svg|png|gif|jpg|json)$/,
       var metricName = match[4];
       var format = match[5];
 
-      var sonarMetricName = metricName;
+      var options = {
+        headers: {
+          Accept: 'application/json'
```
Comment (paulmelnikow) [inline server.js]: You can move the url into `options` too. Take a look at coveralls or codecov, e.g.

**Comment by paulmelnikow** [comment]
Thanks for your contribution. Would like to get this merged. Left one minor comment.

**Comment by kakawait** [comment]
@paulmelnikow sorry for delay... Just pushed

**Comment by paulmelnikow** [comment]
I went ahead and made the `options` change. Thanks!

**Pull request merged**